src/index.js

```diff
@@ -219,7 +219,11 @@ const processResults = ({ data, errors }) => {
     return {
       summary: data
         .map(({ path, url, summary, shortSummary, report }) => {
-          const obj = { summary, report };
+          const extraDataSummary = Object.assign(
+            {},
+            ...summary.map((item) => ({ [item.id]: item.score * 100 })),
+          );
+          const obj = { summary: extraDataSummary, report };
```

Comment:
I'm finding this logic tricky to read. What do you think about using reduce instead?
```suggestion
          const obj = { 
            summary: summary.reduce((acc, item) => {
              acc[item.id] = item.score*100; 
              return acc;
            }, {}), 
            report 
          };
```